wordle-web/src/main.ts

```diff
@@ -22,26 +22,66 @@ const vuetify = createVuetify({
     }
   },
   theme: {
-    defaultTheme: 'dark',
+    defaultTheme: 'darkCalm',
     themes: {
-      dark: {
+      darkCalm: {
+        dark: true,
         colors: {
+          primary: '#1b1b1b',
```

Comment:
it would be cool to see the background colors of the page change along with each of the themes here.
you can do this by adding a  background:"your-color" to the list 🖌️ 🎨